vnet/spec/vnet/core/filters/helpers2.rb

```diff
@@ -0,0 +1,125 @@
+# coding: utf-8
+
+def flow(params)
+  flow_create(
+    table: params[:table],
+    goto_table: params[:goto_table],
+    priority: params[:priority],
+    match_interface: params[:interface_id],
+    cookie: params[:id] | COOKIE_TYPE_FILTER2,
+    match: params[:match]
+  )
+end
+
```

Comment:
What is the reason for this file? Are all these methods used, and why aren't they part of static.rb?